mne/viz/topomap.py

```diff
@@ -273,23 +275,42 @@ def _check_outlines(pos, outlines, head_pos=None):
         # shift and scale the electrode positions
         if 'center' not in head_pos:
             head_pos['center'] = 0.5 * (pos.max(axis=0) + pos.min(axis=0))
-        if 'scale' not in head_pos:
-            # The default is to make the points occupy a slightly smaller
-            # proportion (0.85) of the total width and height
-            # this number was empirically determined (seems to work well)
-            head_pos['scale'] = 0.85 / (pos.max(axis=0) - pos.min(axis=0))
         pos -= head_pos['center']
-        pos *= head_pos['scale']
 
-        # Define the outline of the head, ears and nose
-        outlines = dict()
         if outlines is not None:
-            outlines.update(head=(head_x, head_y), nose=(nose_x, nose_y),
-                            ear_left=(ear_x, ear_y),
-                            ear_right=(-ear_x, ear_y))
+            # Define the outline of the head, ears and nose
+            outlines_dict = dict(head=(head_x, head_y), nose=(nose_x, nose_y),
+                                 ear_left=(ear_x, ear_y),
+                                 ear_right=(-ear_x, ear_y))
+        else:
+            outlines_dict = dict()
+
+        if outlines == 'skirt':
+            if 'scale' not in head_pos:
+                # By default, fit electrodes inside the head circle
+                head_pos['scale'] = 1.0 / (pos.max(axis=0) - pos.min(axis=0))
+            pos *= head_pos['scale']
+
+            # Make the figure encompass slightly more than all points
+            mask_scale = 1.25 * (pos.max(axis=0) - pos.min(axis=0))
+
+            outlines_dict['autoshrink'] = False
+            outlines_dict['mask_pos'] = (mask_scale[0] * head_x,
+                                         mask_scale[1] * head_y)
+            outlines_dict['clip_radius'] = (mask_scale / 2)
```

Comment:
mask_scale is guaranteed to be of type float (line 295), but I'll set it to 2. for readability.